.gitignore

```diff
@@ -6,6 +6,8 @@ target/
 !**/src/main/**/target/
 !**/src/test/**/target/
 
+phase1/AbdulWahab/PythonProject(LoadCSVtoSQL)/__pycache__
+phase1/AbdulWahab/PythonProject(LoadCSVtoSQL)/env
 ### STS ###
```

Comment:
don't add absolute path 
add relative path so that no one else have to add same thing